tulip/gridworld.py

```diff
@@ -326,7 +326,8 @@ def is_reachable(
         @return:
             True if there is a path, False otherwise.
         """
-        # Check input values and handle negative coordinates
+        # Check input values and
```

Comment:
Can you motivate such short lines? I find that readability is less with these.

Reply:
My only concern is having line widths that are different between comments and code. However, I am happy to keep these changes and test usefulness in practice and get feedback from others later (after merging).